core/Enclave/key_operation.cpp

```diff
@@ -97,6 +97,28 @@ static const EVP_MD *GetDigestMode(ehsm_digest_mode_t digestMode)
     }
 }
 
+/**
+ * @brief Get the Sign Verify Message Type for sign/vewrify
+ *
+ * @param message_type raw/digest
+ * @return const ehsm_message_type_t
+ */
+static ehsm_message_type_t GetSignVerifyMessageType(ehsm_data_t *message_type)
```

Comment:
better to convert the type out of enclave, so that we can simplify the edl function parameters, e.g. enum/int.
Can do it in web js?